src/variables.ts

```diff
@@ -0,0 +1,79 @@
+import type { ModuleInstance } from './main.js'
+import type { CompanionVariableValues } from '@companion-module/base'
+
+export function UpdateVariableDefinitions(self: ModuleInstance): void {
+	const variables = []
+	for (let i = 1; i <= 9; i++) {
+		variables.push({ variableId: `awayLineupNumber${i}`, name: `Away Lineup Number ${i}` })
```

Comment:
the individual push function calls can be combined `push(element1, element2, /* …, */ elementN)`